File: example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/NetworkingLogger.java
package com.vimeo.android.networking.example.vimeonetworking;

import android.util.Log;

import com.vimeo.networking.logging.DebugLoggerInterface;

/**
 * Logger delegate for handling logs coming from the Networking library
 * <p/>
 * Created by kylevenn on 1/27/16.
 */
public class NetworkingLogger implements DebugLoggerInterface {

    private static final String TEST_APP = "~NET TEST APP~";

    @Override
    public void e(String error) {
        Log.e(TEST_APP, error);
    }

    @Override
    public void e(String error, Exception exception) {
        Log.e(TEST_APP, error, exception);
    }

    @Override
    public void d(String debug) {
        Log.d(TEST_APP, debug);
    }

    @Override
    public void v(String verbose) {
        Log.v(TEST_APP, verbose);
    }
}

File: example-kotlin-android/src/main/kotlin/com/vimeo/android/networking/example/kotlin/vimeonetworking/NetworkingLogger.kt
package com.vimeo.android.networking.example.kotlin.vimeonetworking

import android.util.Log

import com.vimeo.networking.logging.LogProvider

/**
 * Logger delegate for handling logs coming from the Networking library
 *
 * Created by anthonyr on 5/8/17.
 */
class NetworkingLogger : LogProvider {

    private val TEST_APP = "~NET TEST APP~"

    override fun e(error: String) {
        Log.e(TEST_APP, error)
    }

    override fun e(error: String, exception: Exception) {
        Log.e(TEST_APP, error, exception)
    }

    override fun d(debug: String) {
        Log.d(TEST_APP, debug)
    }

    override fun v(verbose: String) {
        Log.v(TEST_APP, verbose)
    }

}
